packages/xo-server/src/api/proxy.js

```diff
@@ -68,7 +68,7 @@ getAll.permission = 'admin'
 
 export function update({ id, vm, ...props }) {
   return this.updateProxy(id, {
-    vmUuid: vm?.uuid,
+    vmUuid: vm != null ? this.getObject(vm).uuid : vm,
```

Comment:
Use a more explicit value for the else clause.

Reply:
`vm` can be `null` or `undefined`.